src/main/java/at/hannibal2/skyhanni/utils/RenderUtils.kt

```diff
@@ -1267,6 +1267,7 @@ object RenderUtils {
         startAtEye: Boolean = true,
         textSize: Double = 1.0,
         waypointColor: Color =
+            // TODO remove color
```

Comment:
why? You can force it to any colour. The default is just the one I use for the TunnelMaps. Setting it to White for example is effectively the same as removing it (at least I think that is what you mean here)